configure.ac

```diff
@@ -2697,15 +2657,15 @@ yes)
     ;;
 esac
 
-case "$CC_BASENAME" in
-*mpicc*)
+case "$ac_cv_cc_name" in
+mpicc)
```

Comment:
This won't work (we have no MPI compiler name detection), so perhaps we should bring back `$CC_BASENAME` for this switch.

Reply:
3e03fe44f7448dfb67970f13542268d8ae374752 should fix it